src/commands/users/add.ts

```diff
@@ -0,0 +1,46 @@
+import { runCommand } from '../../util/command';
+import { outputFormat } from './util';
+const open = require('open')
+
+export = {
+	command: 'add',
+	desc: 'Add new user',
+	builder: yargs => {
+		yargs.options({
+			'new-user-username': {
```

Comment:
I would make these positional arguments since username, password, display name are required. User group and email are optional parameters.

Reply:
will do